config/eslint/vue-root-class-name.js

```diff
@@ -0,0 +1,171 @@
+"use strict";
+
+const { parse } = require('path')
+const kebabCase = require('lodash/kebabCase')
+
+
+//------------------------------------------------------------------------------
+// Rule Definition
```

Comment:
Could you update the definition here, and give a simple example like in the convention doc?